datadogclient/datadog_client_test.go

```diff
@@ -43,7 +43,7 @@ var _ = Describe("DatadogClient", func() {
 			"test-deployment",
 			"dummy-ip",
 			time.Second,
-			1024,
+			1500,
```

Comment:
Why did you increase this?

Reply:
Adding the additional metrics was causing the "uses tags as an identifier for batching purposes" test to fail because the size of the metric was ~1450 bytes, which meant the client was just throwing them out, so the maximum had to be upped a little.